docs(login,v2): describe cloud usage (#9404)
zitadel/zitadel: docs/docs/guides/integrate/login/hosted-login.mdx

#### Step-by-step Guide

> To try out the new login without modifying it, the easiest way is to just activate it on your Zitadel Cloud instance domain at `https://<YOUR_ZITADEL_INSTANCE_DOMAIN>/ui/v2/login`.
> 
> The easiest way to actually customizing it is to fork the https://github.com/zitadel/typescript repo and use the "Deploy" button to run your code on Vercel.

Author: I agree, visiting is also possible. A bit boring without actually getting access to an app, though :slightly_smiling_face: 

Let's try to be explicit about this with an additional sentence?
```suggestion
**Trying out the new login:** To preview the new login without changing your current setup, the easiest way is to visit `https://<YOUR_ZITADEL_INSTANCE_DOMAIN>/ui/v2/login` on your Zitadel Cloud instance domain. You can also configure Zitadel to redirect users to `https://<YOUR_ZITADEL_INSTANCE_DOMAIN>/ui/v2/login` for authentication.

**Customizing the new login:** The easiest way to actually customizing it is to fork the https://github.com/zitadel/typescript repo and use the "Deploy" button to run your code on Vercel.
```

> The easiest way to actually customizing it is to fork the https://github.com/zitadel/typescript repo and use the "Deploy" button to run your code on Vercel.

Reviewer: The reason I thought it might makes sense to only visit, is so you don't change any of the configuration but can still have a look. So you for sure also don't break anything.

I think with adding the sentence its fine. I would be a little more concrete, or add that its described below in the steps by steps how to do it.

> The easiest way to actually customizing it is to fork the https://github.com/zitadel/typescript repo and use the "Deploy" button to run your code on Vercel.

Author: I added the admonition to where it is most relevant, gave it a better title and specified a little further for what the service account can be used to fix a lock out.

![image](https://github.com/user-attachments/assets/b12f2cfe-4925-40df-a057-eeabd2e2a865)
![image](https://github.com/user-attachments/assets/3243bebd-07bb-4f1c-b44f-344e46369e62)

> The easiest way to actually customizing it is to fork the https://github.com/zitadel/typescript repo and use the "Deploy" button to run your code on Vercel.

Reviewer: yes we can commit

> The easiest way to actually customizing it is to fork the https://github.com/zitadel/typescript repo and use the "Deploy" button to run your code on Vercel.

Author: Done

:::caution
Before you reconfigure Zitadel, we recommend you to create a service user with a personal access token and the IAM_OWNER role for a lockout-failsafe.
:::

<Tabs queryString="deployment">
    <TabItem value="zitadel_cloud_standard" label="Zitadel Cloud Standard V2 Login" default="true">

>         Zitadel Cloud provides the standard V2 login at the fixed path `/ui/v2/login`. To activate it in your Zitadel Cloud instance, you can follow one of these steps:

Reviewer: I guess your intention with the "standard" is to tell that it is not customized. In my opinion hosted login is always about our login version. The word standard leads to more confusion. 

```suggestion
        Zitadel Cloud provides the V2 login at the fixed path `/ui/v2/login`. To activate it in your Zitadel Cloud instance, you can follow one of these steps:
```

    </TabItem>
    <TabItem value="vercel_custom" label="Vercel Custom Login Deployment">

        The simplest way to deploy the new login for yourself is by using the [“Deploy” button in our repository](https://github.com/zitadel/typescript?tab=readme-ov-file#deploy-to-vercel) to deploy the login directly to your Vercel.

        1. [Create a service user](https://zitadel.com/docs/guides/integrate/service-users/personal-access-token#create-a-service-user-with-a-pat) with a PAT in your instance
        2. Give the user IAM_LOGIN_CLIENT Permissions in the default settings (YOUR_DOMAIN/ui/console/instance?id=organizations)
        Note: [Zitadel Manager Guide](https://zitadel.com/docs/guides/manage/console/managers)
        3. Deploy login to Vercel: You can do so by directly clicking the [“Deploy” button](https://github.com/zitadel/typescript?tab=readme-ov-file#deploy-to-vercel) at the bottom of the readme in our [repository](https://github.com/zitadel/typescript)
        4. If you have used the deploy button in the steps before, you will automatically be asked for this step. Enter the environment variables in Vercel

>         - PAT
>         - ZITADEL_API_URL (Example: https://my-domain.zitadel.cloud, no trailing slash)

Reviewer: This are not in sublist anymore. I think this should do the fix (not sure)
```suggestion
           - PAT
           - ZITADEL_API_URL (Example: https://my-domain.zitadel.cloud, no trailing slash)
```

Author: let's try, thx

Author: it works 
![image](https://github.com/user-attachments/assets/c36319c0-5709-49fb-84a5-14145e2f3103)
